cmplibrary/src/test/java/com/sourcepoint/cmplibrary/SourcePointClientBuilderTest.java

```diff
@@ -0,0 +1,54 @@
+package com.sourcepoint.cmplibrary;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+public class SourcePointClientBuilderTest {
+    private SourcePointClientBuilder sourcePointClientBuilder;
+
+    @Before
+    public void setUp() throws ConsentLibException {
+        sourcePointClientBuilder = new SourcePointClientBuilder(808, "siteName", true);//mock(SourcePointClientBuilder.class);
+    }
+
+    private Field getDeclareFieldAccess(String fieldName) throws Exception{
+        Field member = SourcePointClientBuilder.class.getDeclaredField(fieldName);
+        member.setAccessible(true);
+        return member;
+    }
```

Comment:
What if we added the `getDeclaredFieldAccess(String fieldName)` to a helper class/module, so we don't need to re-implement it in every test class? This way we just have to implement it once and import it whenever needed.

Reply:
We should maintain test code of each class in single file.